artiq/test/coredevice/test_moninj.py

```diff
@@ -1,8 +1,28 @@
 import unittest
 import asyncio
+import re
+import numpy
+from contextlib import asynccontextmanager
+from unittest import IsolatedAsyncioTestCase
 
 from artiq.coredevice.comm_moninj import *
 from artiq.test.hardware_testbench import ExperimentCase
+from artiq.experiment import *
+
+
+def async_test(coro):
```

Comment:
For the 238th time please follow existing coding patterns. If you want to introduce (justified) new ones do it consistently throughout the entire codebase.

Reply:
okay, removed